src/calculateRentalCost.js

```diff
@@ -5,6 +5,23 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
```

Comment:
Consider using a constant with a descriptive name for the daily cost value instead of a literal to make the code more readable and maintainable.